libs/sdk-ui-vis-commons/package.json

```diff
@@ -68,7 +68,8 @@
         "lodash": "^4.17.19",
         "react-intl": "^3.6.0",
         "react-measure": "^2.3.0",
-        "tslib": "^2.0.0"
+        "tslib": "^2.0.0",
+        "uuid": "^8.3.2"
```

Comment:
We dont need this reference anymore generating of random id is moved outside of this package